lib/full-site-editing/edit-site-page.php

```diff
@@ -68,7 +77,50 @@ function gutenberg_get_editor_styles() {
 }
 
 /**
- * Initialize the Gutenberg Edit Site Page.
+ * Initialize the Gutenberg Templates List Page.
+ */
+function gutenberg_edit_site_list_init() {
+	wp_enqueue_script( 'wp-edit-site' );
+	wp_enqueue_style( 'wp-edit-site' );
+	wp_enqueue_media();
+
+	$template_type = $_GET['postType'];
```

Comment:
We need to bail here if an invalid `postType` is provided, otherwise a malicious bit of HTML can be output when it is interpolated into the `wp.editSite.initializeList` below.

Fix incoming.